float32 m1FrontVelocity
float32 m2FrontVelocity
float32 m1BackVelocity
float32 m2BackVelocity

float32 m1FrontAmps
float32 m2FrontAmps
float32 m1BackAmps
float32 m2BackAmps

float32 voltageFront
float32 voltageBack

float32 xAccelImu0
float32 yAccelImu0
float32 zAccelImu0
float32 wOrientationImu0
float32 xOrientationImu0
float32 yOrientationImu0
float32 zOrientationImu0

float32 xAccelImu1
float32 yAccelImu1
float32 zAccelImu1
float32 wOrientationImu1
float32 xOrientationImu1
float32 yOrientationImu1
float32 zOrientationImu1